backend/start_server.py

```
@@ -2,39 +2,44 @@
-        subprocess.Popen(["ollama", "serve"], sdout=subprocess.DEVNULL, stderr=subprocess.DEVLNULL)
+        subprocess.Popen(
+            ["ollama", "serve"],
+            stdout=subprocess.DEVNULL,
+            stderr=subprocess.DEVNULL,
+            start_new_session=True,
+            close_fds=True,
+        )
 
-        # Wait for Ollama to start
-        for i in range(30): # Wait up 30s
+        for i in range(30):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)